### DISCUSSION TITLE
Old process of using Smart Naming from the context menu with data highlighting.

### DISCUSSION THREAD

**Discussion by EldrinSMP** [aclap-dev/video-downloadhelper#1105] [Help and Q&A]
### Your question

Be fore the new interface, it was possible it highlight the source portion of the website and right-click, choose SmartNaming from the context menu, and it would create a rule that would name downloaded files based on the area highlighted. It persisted for each page of the site and could be incredibly useful on sites that the downloader could not pull the file name correctly. So far I haven't figured out how to do this with the new UI, and I have no CLUE how to write certain naming rules without it. Is it possible without going back to the old UI? Can it be done?  

Thanks.

### Addon & CoApp version and details

Version 9.1.0.42 (stable), target mozilla, locale en-US.

Companion App installed (v2.0.19) Recheck

/usr/bin/vdhcoapp

**Comment by GrampaWildWilly** [comment]
Well...  You can take a look at #838.  Be sure you click anything in there that says `Show x previous replies`.  There are several examples in there of trying to get smartnaming to work.  They all failed at the time.  But it is my understanding that since we were posting over there, smartnaming has been fixed in the version of VDH you are running.  Give that a try & come back to report your experience.
